### ISSUE TITLE
why my output is like this

### ISSUE THREAD

**Issue by feifeiwei** [lucastabelini/PolyLaneNet#10]
Hello,thanks for your great work,
when i train polyLaneNet with my dataset about 30000 images within about 50 epochs
i run the demo like this, what's the problem with it？

**Comment by lucastabelini** [comment]
How exactly did you train with your dataset? There may be a bug in your dataset loader.

**Comment by feifeiwei** [comment]
my dataloader seem to be right,
How much does your poly loss finally converge to? 
my poly loss is from about 100 to 1~3 within 50 epochs, The question is whether poly loss doesn't converge？

**Comment by lucastabelini** [comment]
On TuSimple the loss converges to ~0.05. You can't just look at the loss value though. If the annotations are not being loaded correctly the model might still converge, but not to what you want. Try visualizing your data after it has been loaded by `LaneDataset`. You can do this by running `python -m lib.datasets.lane_dataset`, which will load the dataset described in the file `config.yaml`.

Since in every image you showed the predictions look the same, it seems the model is converging to something weird, which is generally caused by bad data.

**Comment by feifeiwei** [comment]
There seems to be wrong with my dataset.py script,  thanks for your reply.
